blog/2020-12-21-openhab-3-0-release.md

```diff
@@ -0,0 +1,89 @@
+---
+layout: BlogPost
+category: blog
+title: openHAB 3.0 Release
+author: Kai Kreuzer and Yannick Schaus
+date: '2020-12-21T11:00:00+01:00'
+previewimage: /uploads/winter.jpg
+tags:
+  - news
+excerpt: >-
+  The next major release has arrived with openHAB 3! This is exciting news for our community, as it makes openHAB more modern, simpler and yet more powerful.
+---
+The next major release has arrived with openHAB 3! This is exciting news for our community, as it makes openHAB more modern, simpler and yet more powerful.
+
+<!-- more -->
+
+With openHAB 3 we reach a new level of both functionality as well user-friendliness. Evolving a product with such a large userbase is always a difficult challenge. On the one hand, users wish new features to be shipped and contributors have ideas and want to see their implementations being accepted and integrated. There is hence the continuous risk of a massive feature creep. On the other hand, newbies want a clear path and best practices on how to use the product and having too many different ways and options can easily confuse them as it leads to a steep learning curve.
+
+The balance found for openHAB 3 is really a sweet spot: The underlying framework has been modernized without any negative impact, a lot of complexity has been thrown overboard without dropping features that our users rely on and many things have been streamlined, which culminates in our completely new main user interface.
+
+## Architecture Evolution
+
+Version 1 of openHAB was in itself a clean and simple architecture that worked well up to the moment that more and more "regular" users joined the community and complemented the rather dev-centric crowd.
+
+Version 2 of openHAB had to do compromises between the "old" purely textual openHAB 1 world and the new UI-enabling features that were introduced with Eclipse SmartHome. The way one had to work with bindings and rules very much differed from each other and these two world existed more or less side-by-side, glued together through the 1.x compatibility layer.
+
+A huge step of simplification in openHAB 3 is therefore the removal of the 1.x compatibility layer. All bindings now work exactly the same way and the user decides on the preference of configuration done textually or through the UI. To make sure that nobody is left behind by this move, [many 1.x bindings have been reimplemented](https://github.com/openhab/openhab-addons/issues/6179#migrated) for openHAB 3. For the rest, it is possible to keep an old 2.x instance running while integrating it into a new 3.0 setup - see the [openHAB outposts](#outposts) below.
+
+Furthermore, the two separate rule engine implementations have been merged into one, so that there is now a single concept for automation rules. As a consequence, textual rules written in openHAB's own Rule DSL also turn up within the user interface. 
```

Comment:
```suggestion
Furthermore, the two separate rule engine implementations have been merged into one, so that there is now a single concept for automation rules. As a consequence, textual rules written in openHAB's own Rule DSL are also available within the user interface. 
```